developer-docs-site/docs/guides/resource-accounts/common-questions.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Common questions"
+id: "common-questions"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Common questions
```

Comment:
maybe add some use cases on why we want to create and use a resource account?

Reply:
Good point, maybe went overboard with it but hey, it's useful info!